Contributing.md

```diff
@@ -39,6 +47,7 @@ Current Issue Triagers:
 Our most regular and experienced Issue Triagers sometimes move on to doing code reviews for pull requests, and have input into which pull requests should be merged. 
 
 Current Reviewers:
+- [@hwillson](https://github.com/hwillson)
 - [@lorensr](https://github.com/lorensr)
 - [@abernix](https://github.com/abernix)
```

Comment:
Yup, going to go with this for now.  I think this allows users to easily know who to reach out to for help, and we've added @hwillson who has been a tremendous help lately.